jujugui/static/gui/src/app/d3-components/test-d3-components.js

```diff
@@ -145,20 +145,6 @@ describe('d3-components', function() {
       assert.strictEqual(state.dbldbl, true);
     });
 
-  it('should correctly handle synthetic event bindings', function(done) {
```

Comment:
This test violates isolation and generates errors in subsequent test cases randomly.